Raven.Tests.Helpers/RavenTestBase.cs

```diff
@@ -138,6 +138,24 @@ protected string NewDataPath(string prefix = null, bool forceCreateDir = false)
 			return newDataDir;
 		}
 
+        /// <summary>
+        /// Creates a new Embeddable document store.
+        /// </summary>
+        /// <param name="runInMemory">Whatever the database should run purely in memory. When running in memory, nothing is written to disk and if the server is restarted all data will be lost.<br/>Default: <b>true</b></param>
+        /// <param name="requestedStorage">What storage type to use (see: RavenDB Storage engines).<br/>Allowed values: <b>esent</b>, <b>munin</b>.<br/>Default: <b>esent</b></param>
+        /// <param name="catalog"></param>
```

Comment:
These are custom bundles? How does this differ to `activeBundles`, below?

Link to an example how this is setup (how i would create an instance, which is to be passed in) would be awesome, please.